### PR TITLE
ISSD-1346 [Marketplace] Added Link on Next Steps page

### PR DESCRIPTION
**Pull request by JoMaAlves**

### CODE REVIEW

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:sf

**Review by silasbsi — APPROVED** [review]
Good job @JoMaAlves :martial_arts_uniform:

**Comment by silasbsi** [comment]
ci:forward:force

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/NextStepPage/NextStepPage.tsx [hunk 1]
```diff
@@ -278,16 +274,25 @@ export function NextStepPage({
...
+					{!(
+						paymentStatus === PaymentStatus.PAYMENT_PENDING &&
+						!isTrial
+					) && (
+						<div className="d-flex justify-content-end">
+							<a href="#">
+								<ins>Learn more about App Configuration</ins>
+							</a>
+						</div>
+					)}
```
Comment (claricebrasil) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/NextStepPage/NextStepPage.tsx:295]:
```suggestion
					{!(
						paymentStatus === PaymentStatus.PAYMENT_PENDING &&
						!isTrial
					) && (
						<div className="d-flex justify-content-end">
							<a href="#">
								<ins>Learn more about App Configuration</ins>
							</a>
						</div>
					)}	
```
This text should be displayed only for payment status completed? If so, I believe you could use just
`paymentStatus === PaymentStatus.PAYMENT.PAID`. Trial option comes with the payment status pending too

**Review by claricebrasil — CHANGES_REQUESTED** [review]
Good job, Joma! Just added an observation here ;)

**Comment by silasbsi** [comment]
ci:stop

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/GetAppPage/components/ProductCard/ProductCard.tsx [hunk 2]
```diff
@@ -38,40 +38,26 @@ const ProductCard = ({
...
-			const licenseUsageType = sku?.skuOptions.find(
+	const getProductBasePriceAndTrial = (skus: SKU[]) => {
+		skus?.forEach((sku) => {
+			const licenseUsageTypes = sku?.skuOptions.filter(
 				(skuOption) =>
-					skuOption?.key === 'standard' &&
-					skuOption?.value === 'yes' &&
-					skuOption?.key
+					skuOption?.value === SkuOptions.STANDARD.toLowerCase() ||
+					skuOption?.value === SkuOptions.TRIAL.toLowerCase()
```
[the remark this answers is not in the document]
Comment (JoMaAlves) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/GetAppPage/components/ProductCard/ProductCard.tsx:46]: Is it possible to an app just have a developer option?

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/NextStepPage/NextStepPage.tsx (hunk 1 above)
Comment (JoMaAlves) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/NextStepPage/NextStepPage.tsx:295]: But this message is related to the the app configuration over download. With the Trial option it is possible to download the app

Comment (JoMaAlves) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/NextStepPage/NextStepPage.tsx:295]: Also, over figma, the trial dows have this message

Comment (claricebrasil) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/NextStepPage/NextStepPage.tsx:295]: so, if you just use `paymentStatus === PaymentStatus.PAYMENT_PENDING && isTrial` it doesn't work?

**Pull request closed without merging**